### PR TITLE
CORS allows HTTP connections

### PR DESCRIPTION
**Pull request by MaxWofford**

### CODE REVIEW

File: api/config/initializers/cors.rb [hunk 1]
```diff
@@ -8,7 +8,10 @@
 # Read more: https://github.com/cyu/rack-cors
 
 Rails.application.config.middleware.insert_before 0, Rack::Cors do
-  BASE_ORIGINS = ['https://new.hackclub.com', 'https://hackclub.com'].freeze
+  BASE_ORIGINS = ['https://new.hackclub.com',
+                  'http://new.hackclub.com',
```
Comment (paked) [inline api/config/initializers/cors.rb]: Why? `http://hackclub.com` should automatically redirect to `https://hackclub.com`?

Comment (MaxWofford) [inline api/config/initializers/cors.rb]: https://github.com/hackclub/monolith/issues/85

Comment (paked) [inline api/config/initializers/cors.rb]: @MaxWofford: Interesting... It works for me when I go to `http://hackclub.com` in a private Chrome session.

Comment (MaxWofford) [inline api/config/initializers/cors.rb]: Try Firefox private browsing

Comment (MaxWofford) [inline api/config/initializers/cors.rb]: Ping on this. It's time sensitive for anyone viewing our workshops on http.

**Pull request merged**